### PR TITLE
Begin logging crash site game time to scenario data

### PR DESCRIPTION
**Pull request by Jayefuu**
Added lines to store the unix time the restart happens as well as how many ticks the game lasted. Starting storing these now so I have some interesting data to use in a week or two.

### CODE REVIEW

File: map_gen/maps/crash_site/commands.lua [hunk 1]
```diff
@@ -47,6 +47,11 @@ callback =
             local discord_crashsite_role = '<@&762441731194748958>' -- @crash_site
             --local discord_crashsite_role = '<@&593534612051984431>' -- @test
             Server.to_discord_raw(discord_crashsite_role .. ' **Crash Site has just restarted! Previous map lasted: ' .. time_string .. '!**')
+
+            local unix_time = Server.get_current_time()
+            local game_time = game.ticks_played
+            Server.set_data('crash_site_data', tostring(unix_time),  game_time) -- Store the server unix time as key and total game ticks in the Scenario Data
```
Comment (grilledham) [inline map_gen/maps/crash_site/commands.lua:53]: 'crashsite_data' would be more consistent with the scenario name, but this is minor so it's ok.

Comment (grilledham) [inline map_gen/maps/crash_site/commands.lua:53]: Merely a suggestion, but <https://github.com/Refactorio/RedMew/blob/develop/utils/timestamp.lua#L147> might make the unix time easier to read. Up to you.

**Review by grilledham — APPROVED** [review]

File: map_gen/maps/crash_site/commands.lua (hunk 1 above)
Comment (Jayefuu) [reply map_gen/maps/crash_site/commands.lua:53]: Is it crashsite or crash site? It changes depending on where you look, I had settled on always calling it crash site. Hence the _.

Comment (Jayefuu) [reply map_gen/maps/crash_site/commands.lua:53]: I'll keep it as a long int. Easier for me to do math on later I think. I'm ok with it not being human readable if it makes it easier later.

Comment (grilledham) [reply map_gen/maps/crash_site/commands.lua:53]: Yeah you are right, I'm not very consistent with myself 😄. As players don't see this it doens't really matter.

Comment (grilledham) [reply map_gen/maps/crash_site/commands.lua:53]: That's a good enough reason.

**Pull request merged**